org-roam.el

```diff
@@ -1227,7 +1236,9 @@ If item at point is not Org-roam specific, default to Org behaviour."
          (cons '(file . org-roam--find-file) org-link-frame-setup))
         (let ((inhibit-read-only t))
           (erase-buffer)
-          (unless (eq major-mode 'org-roam-backlinks-mode)
+          (unless (eq major-mode 'org-mode)
+            (org-mode))
+          (unless org-roam-backlinks-mode
             (org-roam-backlinks-mode))
           (make-local-variable 'org-return-follows-link)
```

Comment:
This could in principle also be moved into the mode definition. But see also the comment regarding `org-return` above.